pkg/git/git_suite_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package git_test
```

Comment:
Should be "package git"

Reply:
The package name and this file are automatically generated by running `ginkgo bootstrap` command. So, sorry, I don't think we need to change this one. I keep package name be `git_test`.

Reply:
+1